src/WebJobs.Script.WebHost/App_Start/WebHostResolver.cs

```diff
@@ -138,6 +146,20 @@ private static void EnsureInitialize(WebHostSettings settings)
             }
         }
 
+        private static void ReinitializeAppSettings()
+        {
+            // only in azure environment
+            if (!string.IsNullOrEmpty(Environment.GetEnvironmentVariable("WEBSITE_SITE_NAME")))
```

Comment:
Um ..,  I couldn't find the routine here.  We have one in [kudu](https://github.com/projectkudu/kudu/blob/c0cbe58d9fab04bafd88300590e666f029d47942/Kudu.Core/Environment.cs#L317-L320) but not in webjob sdk.